pages/common/fls.md

```diff
@@ -0,0 +1,16 @@
+# fls
+
+> List files and directories in an image file or device.
+> More information: <https://wiki.sleuthkit.org/index.php?title=Fls>.
+
+- Build a recursive fls list over a device:
+
+`fls -r -m {{original_path}} {{/dev/examplepartition}}`
```

Comment:
Let's give a concrete example instead of "/dev/examplepartition". And also, we usually mention "original" if there is a new destination used in the command. Can you explain that a bit more ?